tracing-subscriber/src/fmt/fmt_subscriber.rs

```diff
@@ -229,18 +229,33 @@ where
     /// - `FmtSpan::NONE`: No events will be synthesized when spans are
     ///    created, entered, exited, or closed. Data from spans will still be
     ///    included as the context for formatted events. This is the default.
-    /// - `FmtSpan::ACTIVE`: Events will be synthesized when spans are entered
-    ///    or exited.
+    /// - `FmtSpan::NEW`: Events will be synthesized when spans are created.
+    /// - `FmtSpan::ENTER`: Events will be synthesized when spans are entered.
+    /// - `FmtSpan::EXIT`: Events will be synthesized when spans are exited.
     /// - `FmtSpan::CLOSE`: An event will be synthesized when a span closes. If
     ///    [timestamps are enabled][time] for this formatter, the generated
     ///    event will contain fields with the span's _busy time_ (the total
     ///    time for which it was entered) and _idle time_ (the total time that
     ///    the span existed but was not entered).
+    /// - `FmtSpan::ACTIVE`: Events will be synthesized when spans are entered
+    ///    or exited.
     /// - `FmtSpan::FULL`: Events will be synthesized whenever a span is
     ///    created, entered, exited, or closed. If timestamps are enabled, the
     ///    close event will contain the span's busy and idle time, as
     ///    described above.
     ///
+    /// The options can be chosen in any combination, for instance, if you
+    /// wanted to emite events when spans are created and closed:
```

Comment:
I _personally_ dislike using the second person in documentation ("you"). How about:

```suggestion
    /// The options can be enabled in any combination. For instance, the following
    /// will synthesize events whenever spans are created and closed:
```